solutions/silver/cf-818F.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: cf-818F
+source: CF
+title: Level Generation
+author: Justin Ji
+---
+
+<Spoiler title="Hint 1">
+
+The upper bound for the number of bridges we can have is $n - 1$. To achieve
+this, we construct a tree.
+
+</Spoiler>
+
+<Spoiler title="Hint 2">
+
+Let's consider allocating a certain amount of these nodes to forming a tree.
+How can we use the rest of these nodes to use as many edges as possible?
+
+</Spoiler>
+
+<Spoiler title="Explanation">
+
+The most bridges we can have in a graph with $n$ nodes is $n - 1$ bridges. For
+those interested, the reason this is the case is that all of our bridges must be part of an
+arbitrary spanning tree of our graph.
```

Comment:
move this to hint 1 - also silver competitors probably don't know what a spanning tree is? you should clarify this 👍

Reply:
done, lmk if its still confusing/misplaced